cmd/distribute/distribute.go

```diff
@@ -101,8 +135,31 @@ func distributeReward() error {
 				return err
 			}
 		}
+
+		// Append bucket information to the list. For using the api to distribute the reward to the bucket.
+		for i := range toAccountAddrList {
```

Comment:
can change to 
```
allToBucketAddressList = append(allToBucketAddressList, toAccountAddrList...)
```

Reply:
The problem is same reason, so that the expanded list cannot record the mapping of representatives and voters and other information.